### ISSUE TITLE
Strange Game Crash

### ISSUE THREAD

**Issue by Tranthula** [Kyle873/DoomRPG#97]
Hey. Just a small thing here, was running along and had this pop up in my console (note, am playing with DRLA and 32 lvl megawad):

ERROR: TRIED TO RESIZE DESTROYED ARRAY POWERMOD

I lost my RLA HUD, and my stats HUD seemed to have moved to a place I had it earlier, but decided to change it. And after a about 10secs, game crashed to desktop. Tried playing the map again but had the game windowed. I still got the same error but with this after it:

WARNING: ACS FUNCTION STRLEN CALLED WITH INVALID STRING ARGUEMENT

I had my HUD and everything was ok. I finished the level and decided to replay the mission from a autosave I had earlier. Fullscreen, immediate crash at the exact same place I had the other errors.
Not sure if a array is supposed to spawn anywhere but a Low Power event (in this case I was NOT in this event), but none-the-less, thought this should be posted.

**Comment by subenji** [comment]
do you happen to have the savefile?
I'll also need a copy of the game log generated from loading it. add:
`logfile log.txt` to autoexec.cfg.

**Comment by Tranthula** [comment]
I have the savefile, but the crash only happens a little ways into the map. I also do have the log for the save.

**Comment by Tranthula** [comment]
Ok, so I've figured, this bug has only happened to me while I'm playing with the Turrent out and about. There is usually no warning as to this happening, so I'll keep giving info as I find out about it.

**Comment by subenji** [comment]
Turrets have a known engine issue related to:
http://forum.zdoom.org/viewtopic.php?f=2&t=52428

We're still waiting for a response on this bug being fixed in engine. I've even submitted the necessary code change and I'm just waiting for them to accept it in:
https://github.com/rheit/zdoom/pull/718

in the meantime, use GZDoom 2.1.1 Stable or if you need 2.2, pre-889 is the latest version before the bug was introduced.
http://devbuilds.drdteam.org/gzdoom/gzdoom-g2.2pre-889-g904a027.7z

If you're already using 2.1.1 Stable please let me know as in that case, this isn't due to this known bug.

**Comment by Tranthula** [comment]
Yes I'm using the 2.1.1 Stable. I'll get the 2.2 and see if I can replicate on that.

**Comment by Tranthula** [comment]
So I did some testing with this, seems like the turret and the Hellstorm Elementals from RLA don't play nice together. Spawned both in both the stable and newest beta for GZDoom. Constant crashing on the stable build, but none on the 2.2 build. Unless I need to do more testing we can probably close this issue.
